### ISSUE TITLE
Build "empty" nginx config when there are no web processes

### ISSUE THREAD

**Issue by josegonzalez** [dokku/dokku#6760]
### Description of feature

Currently, if a user creates an app and any of the following are true:

- either doesn't deploy it
- there are no web processes specified in the Procfile
- there are no running web processes (after a docker restart or similar)

What we should do is create a dummy nginx.conf that null-routes and only shows an error page for users. This way there isn't any ambiguity as to whether the app is created property or not.

Source for this is a [conversation in slack](https://gliderlabs.slack.com/archives/C0FP0R13J/p1711221468151559) where a user created an app and added letsencrypt, but then it wasn't resolving because we don't create a dummy nginx config.

**Comment by bakatz** [comment]
@josegonzalez This sounds like something I ran into recently:

1. Restarted node
2. One app failed to start because its dependency hadn't started yet

=> nginx displayed a default page for this app ("welcome to nginx") while returning 200

I use uptime monitoring that treats a "200" response as OK, so I was never alerted that my app was down.
I was about to file an issue but this looks like the same issue that I ran into 😅

Would be really great to see this one fixed. Even just changing the status code for the default page would be fine.

**Comment by josegonzalez** [comment]
Just a note, I'd also love to see this implemented too! For new features - which this is - I normally only work on them if:

- I'm bored and it interests me
- I'm somehow hitting the issue in my own setup
- It is related to a section of Dokku I am working on for other reasons
- Someone has paid for my time

If you're interested in working on this, feel free to let me know via here (or slack/discord) and I can help walk you through contributing :)

**Comment by bakatz** [comment]
> Just a note, I'd also love to see this implemented too! For new features - which this is - I normally only work on them if:
> 
> * I'm bored and it interests me
> * I'm somehow hitting the issue in my own setup
> * It is related to a section of Dokku I am working on for other reasons
> * Someone has paid for my time
> 
> If you're interested in working on this, feel free to let me know via here (or slack/discord) and I can help walk you through contributing :)

Yep totally understood. I'm happy to contribute, let me know what is necessary to get started and I'll take a look when I have some free cycles. Thanks!

**Comment by josegonzalez** [comment]
Closing as there is a pull request open, this will land in the next minor
